### ISSUE TITLE
Is relaxedjs still under active development?

### ISSUE THREAD

**Issue by rhessing** [RelaxedJS/ReLaXed#187]
It seems that the last code commit was already over a year ago and the issues are not being actively resolved. I wonder if relaxedjs is still under active development?

**Comment by wh81752** [comment]
Appears to be dead, kind of pitty though.

**Comment by zicklag** [comment]
Yeah, looks like there's no activity, but I've used it recently and it still works pretty well, which is a little surprising because JavaScript projects not updated in the last so many years tend to stop working because their dependencies stop building. :smile:

**Comment by zicklag** [comment]
Actually, maybe it would be good to collect a list of alternatives, if there are any, in this issue for people's reference, in case relaxedjs stops working for them or something.

**Comment by DanielRuf** [comment]
Hey everyone,

maintaining big opensource projects for a long time can be very exhausting.
If you are interested to bring ReLaXed forward together, it would be great to see some new PRs.

I can also review and merge them and try to coordinate a new release but in the longterm it would be great to have some people who want to step up and take the new lead under a new name by forking ReLaXed.

According to https://useful-forks.github.io/?repo=RelaxedJS%2FReLaXed there is only https://github.com/wpc009/ReLaXed as fork which has some changes. Maybe these can be integrated.

**Comment by rhessing** [comment]
Hi Daniel, 

Are you working on RelaxedJS by yourself? If so we would definitely need a bigger contributing community because it's a great product and would be a waste if it would not be useable anymore.

I'm not a developer myself but I am willing to help out with smaller code tasks if you want :)

**Comment by DanielRuf** [comment]
No, only @Zulko was the only maintainer and longterm contributor. I was one of the persons who helped with the project at the start.

As a maintainer of other big projects I know how hard and stressful it can be to work on a project alone.

Any help is very welcome and I totally agree with you @rhessing.

**Comment by jobukkit** [comment]
> Actually, maybe it would be good to collect a list of alternatives, if there are any, in this issue for people's reference, in case relaxedjs stops working for them or something.

https://github.com/Kozea/WeasyPrint seems to be an alternative, not sure how they compare though.

**Comment by zicklag** [comment]
It's not an HTML-to-PDF solution, but I've migrated everything from Relaxed to [Typst](https://typst.app/) which is pretty amazing if it fits your use-case.
